lib/src/rmui_bundle_suggestors/constants.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// The script tag for the RMUI bundle.
+const rmuiBundleScript =
+    '<script src="packages/react_material_ui/react-material-ui-development.umd.js"></script>';
```

Comment:
I didn't think about it until now, but I wonder if we should conditionally insert the prod RMU bundle if the matched React JS files have `_prod` in their name...

Reply:
Oh good idea! Yep I can update that!

Reply:
Okay in d53cd06 I added an `isProd` arg to the suggestor constructors so you can specify if the script you're adding is prod or not.